roles/ogm/tasks/script_and_cron.yml

```diff
@@ -34,14 +34,14 @@
      hour: '02'
      minute: '31'
      user: 'umcg-ogm-dm'
-     job: /bin/bash -c '~/make_backup.sh'
+     job: /bin/bash -c '~/make_backup.sh >> ~/make_backup.sh.log'
      cron_file: 'ogm_make_backup'
    - name: 'Clean_OGM_psql_backups  # Unique ID required to update existing cronjob: do not modify.'
      weekday: '*'
      hour: '07'
      minute: '23'
      user: 'umcg-ogm-dm'
-     job: /bin/bash -c '~/manage_psql_backups.sh'
+     job: "/bin/bash -c '~/manage_psql_backups.sh >> ~/manage_psql_backups.sh.log'"
```

Comment:
When we append to a log file in a home dir is there some logrotation / cleanup in place to prevent this from eventually run out of disk space on the machine that executes this cron job?